web/src/components/shared/Toggle.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "../../scss/components/shared/Toggle.scss";
+
+interface Props {
+  items: {
+    onClick: () => void;
+    isActive: boolean;
+    title: string;
+  }[];
+}
+
+export default function Tooltip(props: Props) {
```

Comment:
decided to just make this an FP since it's so small. not really a fan of default exports but just leaving this as is for now

Reply:
also dunno why this the file is Toggle and the component is called Tooltip